packages/src/style.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
Shouldn't we import something here (and in `editor.scss`). Maybe add a commented import to the example block?

Reply:
I've added this comment `//Import editor styles of blocks here`